bin/makef.sh

```diff
@@ -132,8 +150,8 @@ sync
 
 echo "### final fsck, just to be sure"
 fsck.vfat -f -a ${loopback}p2
-fsck.ext4 -f -a -E discard,inode_count_fullmap -D ${loopback}p3
```

Comment:
did you had to remove this ? the inode_count_fullmap makes sure the inode table is completely reordered. Does it cause problems ?

Reply:
I didn't remove it. I simply moved it to the function.

Reply:
oh sorry was blind :-D